walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +313,152 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void customTest1() {
```

Comment:
Could you please add the diagram to the later tests? Are there any more meaningful names for them?

Reply:
Sure, I will add for the later tests. I'm changing the name of tests.